### ISSUE TITLE
Migration pending in v0.6.8

### ISSUE THREAD

**Issue by XaviP** [barseghyanartur/django-fobi#37]
Same problem as v0.6.7, different migration file.
```
python manage.py makemigrations

Migrations for 'fobi':
  0005_auto_20160906_2312.py:
    - Alter field plugin_uid on formelement
    - Alter field plugin_uid on formelemententry
```
0005_auto_20160906_2312.py:
```
# -*- coding: utf-8 -*-
from __future__ import unicode_literals

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [ 
        ('fobi', '0004_auto_20160906_1513'),
    ]   

    operations = [ 
        migrations.AlterField(
            model_name='formelement',
            name='plugin_uid',
            field=models.CharField(verbose_name='Plugin UID', unique=True, max_length=255, editable=False, choices=[(b'boolean', b'Boolean'), (b'checkbox_select_multiple', b'Checkbox select multiple'), (b'content_image', b'Content image'), (b'content_text', b'Content text'), (b'content_video', b'Content video'), (b'date', b'Date'), (b'date_drop_down', b'Date drop down'), (b'datetime', b'DateTime'), (b'decimal', b'Decimal'), (b'dummy', b'Dummy'), (b'email', b'Email'), (b'file', b'File'), (b'float', b'Float'), (b'hidden', b'Hidden'), (b'input', b'Input'), (b'integer', b'Integer'), (b'ip_address', b'IP address'), (b'null_boolean', b'Null boolean'), (b'password', b'Password'), (b'radio', b'Radio'), (b'select', b'Select'), (b'select_model_object', b'Select model object'), (b'select_multiple', b'Select multiple'), (b'select_multiple_model_objects', b'Select multiple model objects'), (b'slug', b'Slug'), (b'text', b'Text'), (b'textarea', b'Textarea'), (b'time', b'Time'), (b'url', b'URL')]),
        ),
        migrations.AlterField(
            model_name='formelemententry',
            name='plugin_uid',
            field=models.CharField(max_length=255, verbose_name='Plugin name', choices=[(b'boolean', b'Boolean'), (b'checkbox_select_multiple', b'Checkbox select multiple'), (b'content_image', b'Content image'), (b'content_text', b'Content text'), (b'content_video', b'Content video'), (b'date', b'Date'), (b'date_drop_down', b'Date drop down'), (b'datetime', b'DateTime'), (b'decimal', b'Decimal'), (b'dummy', b'Dummy'), (b'email', b'Email'), (b'file', b'File'), (b'float', b'Float'), (b'hidden', b'Hidden'), (b'input', b'Input'), (b'integer', b'Integer'), (b'ip_address', b'IP address'), (b'null_boolean', b'Null boolean'), (b'password', b'Password'), (b'radio', b'Radio'), (b'select', b'Select'), (b'select_model_object', b'Select model object'), (b'select_multiple', b'Select multiple'), (b'select_multiple_model_objects', b'Select multiple model objects'), (b'slug', b'Slug'), (b'text', b'Text'), (b'textarea', b'Textarea'), (b'time', b'Time'), (b'url', b'URL')]),
        ), 
    ]
```

**Comment by barseghyanartur** [comment]
@XaviP:

Thanks for checking. I have tracked down the problem. It can't be solved with migrations at all. Instead, I need to fix it on the Django field level. Nothing serious, though. Keep using 0.6.8 in the meantime. I'll make sure it's properly fixed in 0.6.9.

**Comment by XaviP** [comment]
Ok, thanks @barseghyanartur

**Comment by barseghyanartur** [comment]
@XaviP:

Partly fixed in 0.6.9 - I forgot about the form handlers. Migration issue's for the form handlers will be fixed 0.6.10.

**Comment by barseghyanartur** [comment]
@XaviP:

Solved completely in 0.6.10.

**Comment by XaviP** [comment]
Tested, no more migrations after *makemigrations*
Thank you very much @barseghyanartur
